bond_app/fence_token_storage.py

```diff
@@ -105,6 +106,15 @@ def _fetch_and_cache_service_account(self, provider_user, prep_key_fn, fence_fet
 
     def _acquire_lock(self, fsa_key):
         """
+        This thing can return FALSE for 2 different reasons!  And those "FALSE" responses might mean two very different
+        things.  We are currently investigating: https://broadworkbench.atlassian.net/browse/CA-1109
+
+        Either we were unable to successfully lock the FenceServiceAccount record because it is already locked, OR
+        something else happened.  In the latter case, the transaction was successful, but the call to grab the lock
+        still threw an Exception for some reason.  When this happens, do we need to do something different to handle
+        that scenario?  Do we do something different based on the type of the exception that was thrown?  Do we do
+        nothing and just let the exception bubble up?  I think it all depends on the type of Exception that gets thrown,
+        but at the moment, we don't know what those exceptions might be nor what they mean.
```

Comment:
It looks like the comments below cover this case. In any case, I think it is inappropriate to commit debug comments like this.